tools/relay-checker/parameters.go

```diff
@@ -1,26 +1,24 @@
 package main
 
-import (
-	flag "github.com/spf13/pflag"
-)
+import "github.com/iotaledger/hive.go/configuration"
 
-const (
-	// CfgTargetNode defines the config flag of the target node.
-	CfgTargetNode = "relayChecker.targetNode"
-	// CfgTestNodes defines the config flag of the test nodes.
-	CfgTestNodes = "relayChecker.testNodes"
-	// CfgData defines the config flag of the data.
-	CfgData = "relayChecker.data"
-	// CfgCooldownTime defines the config flag of the cooldown time.
-	CfgCooldownTime = "relayChecker.cooldownTime"
-	// CfgRepeat defines the config flag of the repeat.
-	CfgRepeat = "relayChecker.repeat"
-)
+// ParametersDefinition contains the definition of configuration parameters used by the relaychecker.
+type ParametersDefinition struct {
+	// TargetNode defines the target node.
+	TargetNode []string `usage:"the list of nodes to check after the cooldown"`
+	// TestNodes defines test nodes.
+	TestNodes string `default:"http://127.0.0.1:8080" usage:"the target node from the which message will be broadcasted from"`
+	// CfgData defines the data.
+	Data string `default:"TEST99BROADCAST99DATA" usage:"data to broadcast"`
+	// CooldownTime defines the cooldown time.
+	CooldownTime int `default:"10" usage:"the cooldown time after broadcasting the data on the specified target node"`
+	// CfgRepeat defines the repeat.
+	Repeat int `default:"1" usage:"the amount of times to repeat the relay-checker queries"`
+}
+
+// ParametersNetwork contains the configuration parameters of the local peer's network.
+var Parameters = &ParametersDefinition{}
 
 func init() {
-	flag.StringSlice(CfgTargetNode, []string{""}, "the list of nodes to check after the cooldown")
```

Comment:
@jkrvivian should the string be for TestNodes? if so should the TargetNode have the below address?

Reply:
Yes, you are absolutely right!

Reply:
@apenzk I pushed the fixes.